### DISCUSSION TITLE
2021-09-15

### DISCUSSION THREAD

**Discussion by kenyonja** [Dash-Industry-Forum/webRTC#14] [General]
Minutes:
- Review minutes from last meeting: https://github.com/Dash-Industry-Forum/webRTC/discussions/11
- Review draft agenda
  - approved
- Roll Call

<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/tsto/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/tsto/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

Name | Attendee Email | Join Time | Attendance Duration
-- | -- | -- | --
Benard Mesander | ben.mesander@wowza.com | 2021-09-15 07:00:30 | 61 mins
Thomas Stockhammer | tsto@qti.qualcomm.com | 2021-09-15 06:59:56 | 61 mins
Ali C. Begen | q@q.com | 2021-09-15 07:03:16 | 58 mins
Kashyap Kammachi Sreedhar | kashyap.kammachi-sreedhar@nokia.com | 2021-09-15 07:02:50 | 58 mins
Richard Doherty | richard.doherty@dolby.com | 2021-09-15 07:03:18 | 58 mins
Julia Kenyon Phenix | julia.kenyon@phenixrts.com | 2021-09-15 07:03:57 | 57 mins
Mike English | englishm@llnw.com | 2021-09-15 07:06:35 | 55 mins
Piers O'Hanlon | piers.ohanlon@bbc.co.uk | 2021-09-15 07:06:21 | 55 mins
Bill Wishon | bill.wishon@phenixrts.com | 2021-09-15 07:07:10 | 54 mins
Louay Bassbouss Fraunhofer FOKUS | louay.bassbouss@fokus.fraunhofer.de | 2021-09-15 07:07:03 | 54 mins
Nate Burr | nate.burr@verizondigitalmedia.com | 2021-09-15 07:09:39 | 52 mins
Yasser Syed- Comcast | yasser_syed@comcast.com | 2021-09-15 07:10:24 | 51 mins
Phillip Maness Xperi | phillip.maness@xperi.com | 2021-09-15 07:10:21 | 50 mins
Alexander Solonsky castLabs | alexander.solonsky@castlabs.com | 2021-09-15 07:16:38 | 45 mins
Sergio Garcia Murillo | sergio.garcia.murillo@cosmosoftware.io | 2021-09-15 07:19:13 | 42 mins
Michael Stattmann | michael.stattmann@castlabs.com | 2021-09-15 07:04:09 | 27 mins

</body>

</html>

- Action item review
   - Update and refine the scope based on the discussion - In progress: Ali, Thomas
     - closed 
   - Continue to refine the report, adding information - In progress
     - in progress 
   - Update link to draft document on website (https://dashif.org/webRTC/)
     - Thomas will do 
- Document review, moving forward with new version from Ali
   - Link: https://docs.google.com/document/d/1iaWO_oAjUH-Mvrj4bzzD06YENu48RXaZx5DrJOyoT6s/edit#heading=h.h58p5gh4csp0
   -  Review updates to content in clauses 2 (Use Cases), 3 (KPIs), and 4 (Available and Ongoing Work...)
   - Clause 1: 
     - @kenyonja will do an update
     - add some qualification around low-latency and how does it differentiate. Add some context on latency on what it is referred to.
     - @haudiobe will review
   - Clause 2: Use Cases
      - Additional supporting data for latency numbers
      - Any new use cases from Wowza or other contributors
      - @haudiobe add some qualification in 2.6 
      - @haudiobe and piers
      - Benard looks for updates on 2.6
   - Clause 3: KPI
      - Review KPI definitions (if needed)
        - @haudiobe: 
          - change media encoder to distribution encoder and add a note 
          - update some description that there are different instantiations on how many instances are needed for each of the services.
        - change the Live-Edge to time to first frame
        - User interaction
   - Clause 4: Available and Ongoing Work
      - Review updates any any outstanding items
      - Remove section on DASH and WebRTC use cases?
- Next Steps
  - updates as agreed during the call: Julia, Thomas, Ali
  - send out a trigger for review
  - next call will only address new additions and comments

**Comment by kenyonja** [comment]
Is anyone else having issues with the webex link?

**Reply by kenyonja** [reply]
Use the link: https://dashif.my.webex.com/dashif.my/j.php?MTID=m150f82f8e095819d304b8792f17af51b

**Comment by haudiobe** [comment]
I have updated the report with my actions.
